src/geo/leaflet/leaflet-layer-view-factory.js

```diff
@@ -13,30 +13,50 @@ var util = require('../../core/util');
 var MAX_NUMBER_OF_FEATURES_FOR_WEBGL = 10e4;
 
 var LayerGroupViewConstructor = function (layerGroupModel, nativeMap, mapModel) {
-  if (canMapBeRenderedClientSide(mapModel)) {
+  var renderModeResult = getRenderModeResult(mapModel);
+  log.info(renderModeResult);
```

Comment:
We could add a prefix here to make it more clear (otherwise it will just log the object in the console):

```
log.info('Render mode:', renderModeResult);
```